examples/analog_hamiltonian_simulation/09_Noisy_quantum_dynamics_for_Rydberg_atom_arrays.ipynb

```diff
@@ -0,0 +1,3633 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    amplitude_areas = [np.pi/24 * i for i in range(n_times)]
</pre><p>Why 24 and not <code>length * sqrt(2)</code> to show one half oscillation?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>I supposed you meant change <code>np.pi/24</code> to <code>np.pi/(n_times * np.sqrt(2))</code>. Yeah that should work too. Let me stick with these parameters though.   </p>